File: 35-search-insert-position/search-insert-position.js
/**
 * @param {number[]} nums
 * @param {number} target
 * @return {number}
 */
var searchInsert = function(nums, target) {
    if(!nums.find(e => e === target)){ nums.push(target)}
    nums.sort((a,b) => a-b)
    return nums.findIndex(e => e === target) 
};

File: 35-search-insert-position/search-insert-position.java
class Solution {
    public int searchInsert(int[] nums, int target) {
        List<Integer> arrlist 
              = new ArrayList<>();
        
        for (int i : nums) {
            arrlist.add(i);  
        }       
        arrlist.add(target);  
        Collections.sort(arrlist);

         return arrlist.indexOf(target);
    }
}